### PR TITLE
INVALIDATE_DOCKER_CACHE in Bionic and Focal for the next two months

### PR DESCRIPTION
**Pull request by j-rivero**
After many problems related to an apt-key failure that has poisoned many of our docker cache, this PR should invalidate docker cache. This PR will help to keep the cache clean for a couple of months to facilitate image rotation.

See #1069

### CODE REVIEW

File: jenkins-scripts/docker/lib/docker_generate_dockerfile.bash [hunk 1]
```diff
@@ -29,6 +29,14 @@ fi
 
 [[ -z ${INSTALL_C17_COMPILER} ]] && INSTALL_C17_COMPILER=false
 
+# Bionic|Focal builds were affected by a "gpg: keyserver receive failed" in apt-key execution
+# that poisoned a lot of docker cache in different builds and nodes. Force invalidation
+# during a couple of month to rotate images
+if [[ "${DISTRO}" == 'bionic' || "${DISTRO}" == 'focal' ]] && \
+   [[ "$(date '+%s')" -lt "$(date -d '+60 days' '+%s')" ]]; then
```
Comment (scpeters) [inline jenkins-scripts/docker/lib/docker_generate_dockerfile.bash:36]: will this always be 60 days in the future instead of 60 days from a fixed date?

Comment (scpeters) [reply jenkins-scripts/docker/lib/docker_generate_dockerfile.bash:36]: I just expect to see a specific date like `2023-12-31` or `2024-01-10` or something

Comment (j-rivero) [reply jenkins-scripts/docker/lib/docker_generate_dockerfile.bash]: eeeeeeeeeek :sweat: do not code late on Fridays. [33e8fe3](https://github.com/gazebo-tooling/release-tools/pull/1067/commits/33e8fe33af6b23b566264fbe90271296d01a2170) sorry for that

**Comment by j-rivero** [comment]
Testing https://build.osrfoundation.org/view/ign-citadel/job/ignition_citadel-install-pkg-bionic-amd64/804/console.
```
+++ ref_date=2023-11-13
+++ [[ bionic == \b\i\o\n\i\c ]]
++++ date +%s
++++ date -d '2023-11-13+60 days' +%s
+++ [[ 1699890588 -lt 1705017600 ]]
+++ export INVALIDATE_DOCKER_CACHE=true
+++ INVALIDATE_DOCKER_CACHE=true
```
Seems more reasonable to me. Need to merge so we can  test other problems like #1059 .

**Pull request merged**